service/mapper/transitions_internal_test.go

```diff
@@ -551,6 +545,7 @@ func TestTransitions_UpdateTree(t *testing.T) {
 
 		require.NoError(t, err)
 		assert.Equal(t, StatusCollect, st.status)
+		assert.Equal(t, st.updates, updates)
```

Comment:
nit: expected, actual
```suggestion
		assert.Equal(t, updates, st.updates)
```